api/resourcecreator_test.go

```diff
@@ -704,7 +703,7 @@ func TestCreateOrUpdateSecret(t *testing.T) {
 		assert.Equal(t, namespace, secret.ObjectMeta.Namespace)
 		assert.Equal(t, appName, secret.ObjectMeta.Name)
 		assert.Equal(t, []byte(updatedSecretValue), secret.Data["r1_alias_"+secret1Key])
-		assert.Equal(t, updatedFileValue, secret.Data[fileKey1])
+		assert.Equal(t, updatedFileValue, secret.Data["r1_alias_"+fileKey1])
```

Comment:
Since you fixed whitespace in other files. Missing whitespace around `+`.

Reply:
Probably safe to just format the whole file. As it looks good, code wise.

Reply:
`gofmt` is automatically run on save. Is whitespace around `+` your preference or `gofmt`'s?

Reply:
I mean, all changes to the files that are not related to my code is done by `gofmt` when I saved.

Reply:
Oh, I see now that you have removed the whitespace around `+` in previous files. I guess it's my preference then. So we'll let `gofmt` decide then. Even though I think it's harder to read `var1+var2`,  then `var1 + var2`.

Reply:
Yep, I agree on that 👍